config/initializers/deprecations.rb

```diff
@@ -1 +1,30 @@
 ActiveSupport::Deprecation.behavior = :silence if Rails.env.production?
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    unless Foreman.in_rake?
+      if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?('mysql')
+        Foreman::Deprecation.deprecation_warning('1.25', "Support for MySQL is deprecated, please migrate to PostgreSQL")
+        blueprint = NotificationBlueprint.find_by_name('feature_deprecation')
+        return unless blueprint
+        message = UINotifications::StringParser.new(blueprint.message, {feature: 'MySQL as a backend database', version: '1.25'}).to_s
```

Comment:
Other notifications got its classes and define theirs `initialize`, if we want to have a generick deprecation class, wouldn't it great to have such class, which would encapsulate these?

Reply:
Sorry what? ;-) Can you provide an example code?